Assets/Scripts/InteractionController.cs

```diff
@@ -86,6 +88,17 @@ public void HandleRemoteInteraction(Player updatedPlayer, string interactionType
                 // add / move player pin
                 Debug.Log("remote player pinned a location");
                 break;
+            case "annotation":
```

Comment:
Wasn't wild about bringing in the dependency on Colyseus down to here, but since we send all interactions this way without filtering, it felt like the most logical place to extract the most  recent annotation